### PR TITLE
Fix gskan-228. Display org with one Dataset or more

### PR DESCRIPTION
**Pull request by RemiDesgrange**
A bit of refactor has been applied.

### CODE REVIEW

**Comment by fvanderbiest** [comment]
"We can't recompute member" -> does it mean we won't be able to have the correct paging with filtered orgs ?

File: ckanext/pigma_theme/template_helpers.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    """
+    Hack to make helpers.dict_list_reduce work also if provided a dict of dicts instead of a list of dicts
+    """
+    if isinstance(list_, dict):
+        list_ = list_.values()
+    return dict_list_reduce(list_, key, unique)
+
+def filter_orgs(orgs_list):
+    """
+    filter org list of dict based on a critierd, default, orgs with at least one datatset
```
Comment (tonio) [inline ckanext/pigma_theme/template_helpers.py]:
```suggestion
    filter org list of dict based on a critiera: orgs with at least one dataset
```

**Review by tonio — APPROVED** [review]
Untested, code looks good.

**Comment by RemiDesgrange** [comment]
> "We can't recompute member" -> does it mean we won't be able to have the correct paging with filtered orgs ?

It means that once you have created an object `Pager`, mutating the list won't impact the field depending on the list. So we recreate a new object in order to have everything right. On the contrary without it we couldn't have correct paging.

**Pull request merged**